suite-native/atoms/src/Button/Button.tsx

```diff
@@ -12,6 +12,7 @@ export type ButtonColorScheme = 'primary' | 'gray' | 'red';
 
 export interface ButtonProps extends Omit<TouchableOpacityProps, 'style'> {
     iconName?: IconName;
+    iconPosition?: 'left' | 'right';
```

Comment:
Can you try using this type? https://github.com/sindresorhus/type-fest/blob/main/source/require-all-or-none.d.ts

It will enforce that either both of them are needed or none of them. 

Maybe it won't work very well with default values (TS will scream when you don't pass value even though there is default) in which case just leave it as it is now.

Reply:
Good catch @juriczech . Make impossible state unrepresentable.

Reply:
fixed in [30291f2](https://github.com/trezor/trezor-suite/pull/7454/commits/30291f28027b42030eeb6e0b7ab1bc22dddd7f50)